### PR TITLE
(frontend) Set up starter frontend + router

### PR DESCRIPTION
**Pull request by jaza-k**
This branch modifies the ```client``` directory, where I create a basic React application with Typescript support, integrated TailwindCSS for styling as well as a simple React Router implementation

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "@types/node": "^16.18.2",
+    "@types/react": "^18.0.24",
+    "@types/react-dom": "^18.0.8",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.4.2",
+    "react-scripts": "^5.0.1",
+    "tailwindcss": "^3.2.1",
+    "typescript": "^4.8.4",
+    "web-vitals": "^2.1.4"
```
Comment (cloudyyoung) [inline client/package.json:19]: Looks like we're missing Axios here.
Axios is a popular library for sending http requests.

**Review by cloudyyoung — APPROVED** [review]
LGTM! One comment for a missing library.

**Review by tim-macphail — APPROVED** [review]

**Comment by cloudyyoung** [comment]
Checks are failing for this pr, wonder what's going on.

**Comment by tim-macphail** [comment]
> Checks are failing for this pr, wonder what's going on. @tim-macphail

You resolved this @cloudyyoung ?

**Comment by jaza-k** [comment]
Happened to me as well earlier, pretty sure it's a result of Tim's commits not being signed off properly. In this case I just decided to override the check, but in the future you can add the ```-s``` flag to your commits:

Screenshot from 2022-10-28 13 02 13

Or if using the Github web editor:

Screenshot from 2022-10-28 13 02 37

**Comment by cloudyyoung** [comment]
Not sure if sign-off is really necessary for our project, what do you think? @tim-macphail

**Comment by tim-macphail** [comment]
They are mandated by the organization (techstartucalgary) unfortunately. We'll just have to add the `-s` flag on each commit for now @cloudyyoung

**Pull request merged**